theano/gpuarray/multinomial.py

```diff
@@ -450,12 +464,13 @@ def c_code(self, node, name, inp, outputs, sub):
         }
         if(%(sync)d)
             GpuArray_sync(&(out->ga));
+        Py_DECREF(pvals_copy);
```

Comment:
Good catch, but there are other exit points before this where we should free it too.